src/renderer/coremods/installer/util.ts

```diff
@@ -246,6 +246,17 @@ export async function installFlow(
 
   await install(info);
 
+  if ("plaintextPatches" in info.manifest) {
+    const res = await modal.confirm({
```

Comment:
Can you make this not block the function return?

Reply:
I guess I could try something with events?